### ISSUE TITLE
Currently no user-friendly installer for any platform.

### ISSUE THREAD

**Issue by DarkLinkXXXX** [FellowTraveler/Open-Transactions-old#117]
There is currently no user-friendly gui installer for any platform. As is, open-trasactions is still experimental, so maybe for now we want to keep it only developer friendly. However, I believe that if this project is ever to gain traction, either an install for unix and windows is needed, or a cross platform install (lzpack seems good at that).

**Comment by jspit** [comment]
hi

that is the problem i have - i tried to install on windows 7 and it is not
working

thanks
jesmond

On 5 October 2012 01:04, DarkLinkXXXX notifications@github.com wrote:

> There is currently no user-friendly gui installer for any platform. As is,
> open-trasactions is still experimental, so maybe for now we want to keep it
> only developer friendly. However, I believe that if this project is ever to
> gain traction, either an install for unix and windows is needed, or a cross
> platform install (lzpack seems good at that).
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/FellowTraveler/Open-Transactions/issues/117.

## 

Jesmond Spiteri

**Comment by ghost** [comment]
There is a package for gentoo in the bitcoin overlay
https://gitorious.org/bitcoin/gentoo/trees/master/app-crypt/opentxs

USE="git" emerge -u layman && layman -a bitcoin && emerge -av opentxs

Of course, that's not very user-friendly...

**Comment by ghost** [comment]
Yeah, it is pretty significant that we get a user friendly installer. Takes forever to get the pre-requites and install only to find it doesn't work. Would appreciate it if this was finally be distributed.
